libs/designer-ui/src/lib/index.ts

```diff
@@ -16,7 +16,10 @@ export * from './card/addActionCard';
 export { ChatInput } from './chatbot/chatInputBox';
```

Comment:
instead of all these files that we export, we can create a top-level index file and export all of the components there